org.adempiere.base/src/org/compiere/model/MCost.java

```diff
@@ -94,6 +94,33 @@ public static BigDecimal getCurrentCost (MProduct product,
 		MAcctSchema as, int AD_Org_ID, String costingMethod,
 		BigDecimal qty, int C_OrderLine_ID,
 		boolean zeroCostsOK, String trxName)
+	{
+		return getCurrentCost(product, M_AttributeSetInstance_ID, as, AD_Org_ID, costingMethod, qty, C_OrderLine_ID, zeroCostsOK, null, null, false, trxName);
+	}
+	
+	/**
+	 * Retrieve/Calculate Current Cost Price
+	 * @param product product
+	 * @param M_AttributeSetInstance_ID asi for batchlot costing level
+	 * @param as schema
+	 * @param AD_Org_ID org for organization costing level
+	 * @param costingMethod AcctSchema.COSTINGMETHOD_*
+	 * @param qty qty
+	 * @param C_OrderLine_ID optional order line
+	 * @param zeroCostsOK zero/no costs are OK
+	 * @param dateAcct account date
+	 * @param history cost history record
+	 * @param isPostProcess post process

```

Comment:
It is isBackDateProcess in ProductCost, please use consistent variable naming everywhere. Such inconsistency is a bad practise.

Reply:
I think isBackDateProcess sounds clearer.

Reply:
Renamed to isBackDateProcess